### ISSUE TITLE
Basic clone now seems to require MySQL not sqlite

### ISSUE THREAD

**Issue by SuziTech** [imwilsonxu/fbone#42]
Hiya, just cloned fbone and followed next instruction: 'fab setup'.  This blew-up complaining about MySQL.  I don't have MySQL on my wee laptop.  The installation guide said fbone relies on sqlite which I do have.  I've looked in some of the files but can't see where MySQL is set as the db so I can set it back to sqlite.  Do you not support a sqlite version anymore?  How can I modify the setup so it uses sqlite again?
Many thanks,
Suzi.

**Comment by thoppe** [comment]
I seem to be running into the same problem on a stock Ubuntu 14.04 install. Sqlite is installed but `fab setup` gives:

```
  File "/tmp/easy_install-hBY9lC/MySQL-python-1.2.5/setup.py", line 17, in <module>
    install_requires=[
  File "/tmp/easy_install-hBY9lC/MySQL-python-1.2.5/setup_posix.py", line 43, in get_config
  File "/tmp/easy_install-hBY9lC/MySQL-python-1.2.5/setup_posix.py", line 25, in mysql_config
EnvironmentError: mysql_config not found

Fatal error: local() encountered an error (return code 1) while executing 'python setup.py install'
```

**Comment by thoppe** [comment]
For what it's worth a quick search found this [SO question](http://stackoverflow.com/a/5178698/249341) give me a working solution:

    sudo apt-get install libmysqlclient-dev

**Comment by bilalel** [comment]
Or you can simply comment the line with mysql-python, in requirements fabric file.

Bilal

**Comment by imwilsonxu** [comment]
Fix with 6ff899e7e8f8d5de7e55a0b8cb1ec749192798d7
